src/services/users.js

```diff
@@ -19,10 +23,17 @@ const createNewUser = async (user) => {
 		address
 	] = user
 	const passwordHash = await genPasswordHash(password)
-	// eslint-disable-next-line max-len
-	const { rows, error } = await db.query(
-		// eslint-disable-next-line max-len
-		'INSERT INTO users ("firstName", "lastName", "email", "passwordHash", "gender", "jobRole", "department", "address") VALUES ($1, $2, $3, $4, $5, $6, $7, $8) RETURNING *',
+	const refreshToken = crypto.randomBytes(127, (err, buf) => {
```

Comment:
not sure this returns the refreshToken before  the next line runs

Reply:
kindly look into the asynchronous version where you can await

Reply:
also use `40` instead of `127`, the max character limit you set for the refresh token in the database is just `100` so not sure this would work